sysdump/sysdump.go

```diff
@@ -537,23 +537,6 @@ func (c *Collector) Run() error {
 				return nil
 			},
 		},
-		{
-			CreatesSubtasks: true,
-			Description:     "Collecting Hubble flows from Cilium pods",
-			Quick:           false,
-			Task: func(ctx context.Context) error {
-				p, err := c.client.ListPods(ctx, c.options.CiliumNamespace, metav1.ListOptions{
-					LabelSelector: c.options.CiliumLabelSelector,
-				})
-				if err != nil {
-					return fmt.Errorf("failed to get Cilium pods: %w", err)
-				}
-				if err := c.submitHubbleFlowsTasks(ctx, filterPods(p, nodeList), ciliumAgentContainerName, absoluteTempPath); err != nil {
-					return fmt.Errorf("failed to collect hubble flows: %w", err)
-				}
-				return nil
-			},
-		},
```

Comment:
Where is this duplicated?

Reply:
https://github.com/cilium/cilium-cli/blob/e0edad13f37171260cf4f399d4793e2a0b4ea3c8/sysdump/sysdump.go#L660 😱